unittests/output/expected_basic.json

```diff
@@ -0,0 +1,611 @@
+{
```

Comment:
I recommend renaming this file to `expected_good_basic.json` and similar renaming to the rest of the files to maintain the naming convention